acceptance/indy_acceptance/test/client.py

```diff
@@ -1,60 +0,0 @@
-import json
```

Comment:
Did we remove it intentionally?

Reply:
Yes. This is obsolete tests which were written before https://github.com/hyperledger/indy-post-install-automation